### ISSUE TITLE
Support for ipad

### ISSUE THREAD

**Issue by cschaba** [warpdotdev/Warp#2083]
### Discord username (optional)

Carsten#7353

### Describe the solution you'd like?

```text
Hi I would think warp is ideal for use on iPad.  

With all the completions etc.

Maybe there could be some synergies with https://github.com/blinksh/blink (my current favorite Terminal App on the iPad) or https://secureshellfish.app ?

Great job so far! Thanks.

Carsten
```

### How important is this feature to you?

4

**Comment by dannyneira** [comment]
Thanks for this feature request! Please +1 this request as it helps us gauge interest. We will post any updates here.

**Comment by clarkezone** [comment]
+1.  I currently use blink and la terminal on iPad but would love Warp there.

**Comment by obelixA** [comment]
+1

**Comment by tkgreg** [comment]
I’m ready to buy it for 10$ per year. Looks like a good deal for such amazing product. M1 support should be enough and easy to port.

**Comment by dannyneira** [comment]
Part of our roadmap is to support Warp on the WebAssembly, so if iPad browsers support it may be possible in the future?! 🤞

**Comment by clarkezone** [comment]
Please not in the browser.  I wouldn't use that.

**Comment by pmahdavii** [comment]
+1

**Comment by dkauper** [comment]
👍

**Comment by ivanlei** [comment]
+1

**Comment by paprins** [comment]
+1

**Comment by adoolaard** [comment]
+1

**Comment by andrew-manger** [comment]
+1

**Comment by bjurecska** [comment]
This would be amazing. I don't think there is anybody that wouldn't pay a subscription even for having warp on ipad.

**Comment by andrew-manger** [comment]
> This would be amazing. I don't think there is anybody that wouldn't pay a subscription even for having warp on ipad.

I wouldn't pay a subscription for a terminal, that's bs.

**Comment by pure-js** [comment]
+1

**Comment by pure-js** [comment]
> > This would be amazing. I don't think there is anybody that wouldn't pay a subscription even for having warp on ipad.
> 
> 
> 
> I wouldn't pay a subscription for a terminal, that's bs. 

Existing iOS terminals already ask for a subscribtion - usually around 22$ a year, but they are not even close to the WARP

**Comment by andrew-manger** [comment]
> > > This would be amazing. I don't think there is anybody that wouldn't pay a subscription even for having warp on ipad.
> > 
> > 
> > I wouldn't pay a subscription for a terminal, that's bs.
> 
> Existing iOS terminals already ask for a subscribtion - usually around 20$ a year, but they are not even close to the WARP

And that's exactly why I don't use them 😂 I like to own my stuff so I'm pretty biased against subscriptions on principal

**Comment by bjurecska** [comment]
@andrew-manger I am not saying a subscription mode is for everybody, but if you had the choice of buying a subscription and having warp on an iPad pro or not have it at all, then it would suit everybody. You could still chose to not buy it, and other people who are willing to pay the money could have a chance to use it on an iPad pro. Anyways warp is already a subscription based terminal. Maybe you use the free version, which is cool but there is already features that require payments. The single payment based model to outright own your stuff does not bring good quality software as you would want to be making sure that the software you bought keeps getting fixes and new features and it's not reasonable to assume they will do that for free, as it costs them money to pay for the developers who make those features. I pay for gitkraken monthly as well and I love the simplicity it provides. I feel the same way about warp, ever since I use it, my workflow simplified by a lot, and I am willing to pay monthly for that, and I believe I am not alone in this regard.

**Comment by hsteinshiromoto** [comment]
+1

**Comment by davidovari** [comment]
+1

**Comment by Tristan747** [comment]
+1

**Comment by luman75** [comment]
+1

**Comment by Nazarovv676** [comment]
+1

**Comment by lslayer** [comment]
+1

**Comment by andreu4448** [comment]
+1

**Comment by amrshakya** [comment]
+1

**Comment by luofang34** [comment]
+1

**Comment by Hooooot13** [comment]
+1 this would be rad

**Comment by mamrhein** [comment]
+1

**Comment by pabumake** [comment]
+1

**Comment by r351574nc3** [comment]
+1 💯 👍

**Comment by r351574nc3** [comment]
With M4 ipads available now, this is that much more important

**Comment by jmsfez** [comment]
+1

**Comment by feitoria-hugo** [comment]
+1

**Comment by fr0der1c** [comment]
+1

**Comment by dannyneira** [comment]
To anyone else interested in this feature, please add a 👍 to the [original post at the top](https://github.com/warpdotdev/Warp/issues/2083#issue-1444346170) to signal that you want this feature, and subscribe if you'd like to be notified.

Please avoid spam comments like "+1".

**Comment by WilfridoT** [comment]
I would like too

**Comment by debug400** [comment]
👍

**Comment by jcprz** [comment]
👍🏽

**Comment by KrumValkov** [comment]
+1

**Comment by kaizenjinco** [comment]
+1

**Comment by gabeosx** [comment]
Yes please

**Comment by ferrisz** [comment]
+1

**Comment by WilfridoT** [comment]
+1

**Comment by ytwytw** [comment]
+1

**Comment by jack-arturo** [comment]
+1

**Comment by MorenoLaQuatra** [comment]
+1

**Comment by cschaba** [comment]
@RokeJulianLockhart is it possible in Github that you then take over this task?

**Comment by RokeJulianLockhart** [comment]
@cschaba, no, and in *that* context, I understand. If you <del>re-open this issue, then</del> *close it as a duplicate* of [`issues/7495`](https://github.com/warpdotdev/Warp/issues/7495), you can, in effect, hand-off responsibility to me. *Once done*, if notifications are why you don't want to be the owner of this issue, see the undermentioned:

<p></p><details><p></p>

You can customise your subscription to your own issue on the bottom-right, where you can disable it:

1. <p></p><img width="592" height="220" alt="Image" src="https://github.com/user-attachments/assets/ce7c548e-556a-41c9-8b87-26b8091ee17f" /><p></p>

1. <p></p><img width="960" height="520" alt="Image" src="https://github.com/user-attachments/assets/e1cfff87-b0b2-49c9-9b32-21ba1db3db51" /><p></p>

<p></p></details><p></p>

**Comment by scottrfrancis** [comment]
+1000

**Comment by ijmiddleton** [comment]
+1!

**Comment by RokeJulianLockhart** [comment]
@dannyneira, can you re-triage this as [`#issuecomment-3306811791`](https://github.com/warpdotdev/Warp/issues/2083#issuecomment-3306811791) describes, to dissuade additional comments here? I'd like to avoid locking this, so that comments remain editable, etcetera.

**Comment by mohd-saaduddin-ansari** [comment]
+1

**Comment by kevinisyoung** [comment]
+10000

**Comment by artmen1516** [comment]
+1

**Comment by CINC0S** [comment]
Even a warp client would be ideal. A solution similar to the Codex Relay would be beneficial, would be nice to have SSH access and the ability to send agentic tasks.
